apps/design-system/src/subjects/views/execution/pipeline-execution-graph.tsx

```diff
@@ -60,9 +69,36 @@ export function StepNodeComponent({
 }: {
   node: LeafNodeInternalType<StepNodeDataType>
 } & NodeProps) {
-  const { name, icon } = node.data
+  const { name, icon, logs, stage } = node.data
 
-  return <PipelineNodes.StepNode name={name} icon={icon} onEllipsisClick={() => undefined} readonly={readonly} />
+  return (
+    <Drawer.Root direction="right">
```

Comment:
we should render the drawer only in the "execution" mode of the pipeline graph

Reply:
Done.